actions/executeInstructions.ts

```diff
@@ -20,9 +20,14 @@ export const executeInstructions = async (
   multiTransactionMode = false
 ) => {
   const instructions: TransactionInstruction[] = []
+  const copyOfProposalInstructions = [...proposalInstructions]
```

Comment:
we can just sort `proposalInstructions` in place, without created a copy, is that right?

Reply:
would suggest that you make a PR for this directly to Gov UI, since its a generic fix